content/sips/sip-161.md

```diff
@@ -48,72 +50,155 @@ On the topic of scaling and decentralizing the Synthetix Protocol, the onboardin
+For the long-term sustainability of the Synthetix Protocol, the Core Contributor Committee will be responsible for collecting and discussing ideas for the long-term vision of the Synthetix Protocol. This means that the CCC will be responsible for interfacing with all the Synthetix Governing Bodies and Stakeholders to help synthesize a direction in which the Protocol should aim to follow.
+
+#### Core Contributor Committee Stipends & Transparency
+
+The Core Contributor Committee will be a incentivized role funded by the Treasury Council, the CCC stipends amounts will be openly shared. However, the amounts decided on by the Committee in regards to new and existing Core Contributors will remain accessible by only those involved internally.
 
 ### Rationale
 
 <!--This is where you explain the reasoning behind how you propose to solve the problem. Why did you propose to implement the change in this way, what were the considerations and trade-offs? The rationale fleshes out what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.-->
 
 The Core Contributor Committee ratification has been formed from a variety of considerations
 
-_Voting System_ the CCC members are voted in by the Core Contributors as they are the ones who are directly influenced by the decisions that are being made, the Core Contributors are also selected because they are the ones who have the most context making the relationship between the Core Contributors and CCC members symbiotic and accountable.
+_Voting System_
+
+The CCC members are voted in by the Core Contributors as they are the ones who are directly influenced by the decisions that are being made, the Core Contributors are also selected because they are the ones who have the most context making the relationship between the Core Contributors and CCC members symbiotic and accountable.
+
+_Core Contributor Engagement_
+
+The Core Contributor Committee is in the best position to understand which resources are lacking within the Core Contributors and the budget capacity while the existing Core Contributors themselves understand which candidates would be most suitable for the role they are applying for, this vouching system ensures that both parties are responsible for what they know best. This also applies for the disengagement of Core Contributors, since this Committee will be regularly interfacing with the CC's they will have the best understanding of performance.
+
+_Core Contributor Compensation_
+
+The CCC have full discretion over the budget provided by the Treasury Council so they are able to ensure that the Core Contributors incentives are aligned with the Synthetix Protocol vision.
```

Comment:
Change "that the Core Contributors incentives" to "that the Core Contributor incentives"